shell/platform/darwin/macos/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -482,7 +512,16 @@ - (BOOL)handleKeyEvent:(NSEvent*)event {
     return NO;
   }
 
-  return [_textInputContext handleEvent:event];
+  _eventProducedOutput = NO;
```

Comment:
By application do you mean the framework? Because if so that's already happening. On the other hand If you mean the rest of cocoa responder chain, then I'd say no - this would be completely different behavior that what happens without text input (framework first, cocoa second).

Reply:
The whole purpose of this PR is to make sure that TextInputContext doesn't doesn't simply swallow keyEquivalent events that don't produce any text editing action so that they are forwarded to the rest of responder chain.

Reply:
> So it's already "framework first"?

It is. This is the main keyboard event processing:
https://github.com/flutter/engine/blob/0cc828c9b611316ee18b96e01b29c23ba126dd56/shell/platform/darwin/macos/framework/Source/FlutterKeyboardManager.mm#L198-L229

Primary responders are `FlutterEmbedderKeyResponder` and `FlutterChannelKeyResponder` (framework). If these don't handle keyboard event, the event is passed to [`[FlutterKeyboardManager dispatchTextEvent:]`](https://github.com/flutter/engine/blob/0cc828c9b611316ee18b96e01b29c23ba126dd56/shell/platform/darwin/macos/framework/Source/FlutterKeyboardManager.mm#L231). This will let the `TextInputPlugin` process event, and if not handled will forward it to `nextResponder`. 

The issue this PR fixes is that any event sent to TextInputPlugin was marked as handled, even if it was keyboard shortcut that didn't result in any action.

Reply:
Yes, `FlutterTextInputPlugin#handleKeyEvent` is called only when the Framework's keyboard systems choose not to handle the event.